crowbar_framework/app/models/crowbar_service.rb

```diff
@@ -307,6 +307,7 @@ def disable_non_core_proposals
   def prepare_nodes_for_os_upgrade
     upgrade_nodes = NodeObject.all.reject { |node| node.admin? || node[:platform] == "windows" }
     admin_node = NodeObject.admin_node
+    upgrade_nodes_failed = []
```

Comment:
Then we can move it in the future for now we follow the common ruby style. We decided to not merge easy fixable style issues.